trendmicro_deepsecurity/plugin.spec.yaml

```
@@ -0,0 +1,99 @@
+version: 1.0.0
+vendor: philipp-behmer
+support: community
+status: []
+enable_cache: false
+resources:
+  source_url: https://github.com/rapid7/insightconnect-plugins/tree/master/trendmicro_deepsecurity
+  license_url: https://github.com/rapid7/insightconnect-plugins/blob/master/LICENSE
+  vendor_url: https://trendmicro.com
+tags:
+- vulnerability_scanning
+- vulnerability_management
+- intrusion_detection
+- intrusion_protection
+- virtual_patching
+- remediation
+- trend_micro
+- deep_security
+hub_tags:
+  use_cases: [vulnerability_management, cloud_security, remediation_management, threat_detection_and_response ]
+  keywords: [vulnerability, scanning, management, remediation, virtual, patching, intrusion, detection, prevention, remediation, trend, micro, deep, security, trendmicro, deepsecurity]
+  features: []
+connection:
+  dsm_url:
+    title: Deep Security Manager URL
+    description: URL of the Deep Security Manager
+    type: string
+    default: https://app.deepsecurity.trendmicro.com
+    required: true
+  dsm_api_key:
+    title: Deep Security Manager API Key
+    description: API key of the Deep Security Manager
+    type: credential_secret_key
+    required: true
+actions: 
+  search_rules:
+    title: Search
+    description: Search Deep Security IPS rules for given CVEs
+    input:
+      vulnerabilities:
+        title: Vulnerabilities
+        type: "[]string"
+        description: CVEs to protect against
+        required: true
+    output:
+      ips_rules:
+        title: IPS Rules
+        description: IPS rules matching the given CVE numbers
+        type: "[]integer"
+        required: false
+      matched_cves:
+        title: Matched CVEs
+        description: CVEs that have at least one matching IPS rule in Deep Security
+        type: "[]string"
+        required: false
+      missed_cves:
+        title: Missed CVEs
+        description: CVEs without matching IPS rule in Deep Security
+        type: "[]string"
+        required: false
+  deploy_rules:
+    title: Deploy
+    description: Deploy rules in Deep Security
+    input:
+      computer_or_policy:
+        title: Target
+        type: string
+        description: Define if the rules get assigned to a computer or policy
+        required: true 
+        enum:
+          - computer
+          - policy 
+      id:
+        title: ID
+        type: integer
+        description: ID of the target asset or policy in Deep Security
+        required: true
+      rules:
+        title: IPS Rules
+        type: "[]integer"
+        description: IPS rules to assign
+        required: true
+    output:
+      rules_assigned:
+        title: Rules Assigned
+        description: All rules that are currently assigned to the asset
+        type: "[]integer"
+        required: false
+      rules_not_assigned:
+        title: Not Assigned Rules
+        description: Rules that were not assigned
```

```suggestion
        description: Unassigned rules
```
Thoughts? Trying to keep descriptions succinct